tests/data/power_flow/asym-line/rmatrix-with-n-xmatrix-with-n-c1-c0/test.dss

```diff
@@ -0,0 +1,26 @@
+Clear
+
+Set DefaultBaseFrequency=50
+
+! Swing or Source Bar
+New circuit.test_1 phases=3 pu=1.0 basekv=10 bus1=Node1.1.2.3.4
```

Comment:
```suggestion
New circuit.test_1 phases=3 pu=1.0 basekv=10 bus1=Node1.1.2.3.4 MVAsc3=1000000000 MVAsc1=1000000000.0 x1r1=0.166666 x0r0=0.166666
```
For here and all cases. This would be corresponding to sk=1e15 and rx_ratio=6.0